packages/app/src/test-utils.tsx

```diff
@@ -81,6 +81,10 @@ export async function renderApp(
 
     selectVisTab: async (name) => {
       await user.click(await screen.findByRole('tab', { name }));
+
+      if (!withFakeTimers) {
+        await waitForAllLoaders();
+      }
```

Comment:
What is this for ?

Reply:
Ah, so far we were using `selectVisTab` only in `VisSelector.test.ts`, to assert that the selected tab had changed. Now we're using it to test that the selected visualization renders correctly, so we need to wait for the data to load, like we do in `selectExplorerNode`.